themes/hello-friend-ng/layouts/index.html

```diff
@@ -63,11 +62,19 @@ <h1 class="d_text_display d_span_opacity comic-neue"><span id="dynamic_d">D</spa
             </div>
         </div>
     </div>
+    {{ if ne (i18n "finish[translation]") "yes"}}
+      {{ if ne (.Lang) "en"}}
+        <div class="alert alert-warning" role="alert" style="position: absolute; top: 80px; z-index: 1; width: 100%">
+          This is not yet full translated to {{ if eq (.Lang) "pt-pt"}}<span class="flag flag-icon flag-icon-pt"></span>{{ else }}<span class="flag flag-icon flag-icon-{{ index $.Site.Data.langFlags (.Lang) }}" ></span>{{ end }} <span class="badge bg-dark text-light comic-neue">{{.Lang}}</span>. Please help us translating going here: <a href="https://github.com/dogecoinfoundation/dogecoin.com/blob/main/doc/how-to-translate-website.md"><b>How to Translate</b></a>

```

Comment:
Consider changing "This is not yet full translated to" to "This is only partially translated to".

Consider changing "Please help us translating going here:" to "Please help us finish the translation here:"

:)